client/routes/namespace/workflow-archival/components/archival-disabled-messaging/index.vue

```diff
@@ -16,17 +16,25 @@
       <p>
         {{ historyArchivalDisabledMessage }}
       </p>
-      <div v-if="historyArchivalEnableCommand">
-        <pre>{{ historyArchivalEnableCommand }}</pre>
-      </div>
+      <code lang="bash" v-if="historyArchivalEnableCommand">{{
+        historyArchivalEnableCommand
+      }}</code>
     </div>
     <div v-if="!isVisibilityArchivalEnabled" class="message-group">
       <p>
         {{ visibilityArchivalDisabledMessage }}
       </p>
-      <div v-if="visibilityArchivalEnableCommand">
-        <pre>{{ visibilityArchivalEnableCommand }}</pre>
-      </div>
+      <code lang="bash" v-if="visibilityArchivalEnableCommand">{{
+        visibilityArchivalEnableCommand
+      }}</code>
```

Comment:
You may want to omit any mention of archiving Visibility records - right now those do not even work AFAIK.

The new guide I am writing covers that detail - but the current one does not, and I don't have an ETA of when the new one will be published at this point.

Reply:
the request on archival page will fail if visibility is disabled. We could provide an option to opt out from having archival view

Reply:
Perhaps having it there will incentivize the team to get it working ;)